docs/src/pages/versions/versions.md

```diff
@@ -39,16 +56,17 @@ In general, you can expect the following release cycle:
 
 - A **major** release every 12 months.
```

Comment:
I think we're contradicting ourselves here. We say that the user can expect a major release every 12 months, but later in the release schedule, there's not one release that followed this (and v6 won't follow either). 

```suggestion
- A **major** release at least every 18 months.
```

BTW, I think the page as a whole works well for MUI X too, but perhaps we need to split it here to show the "release frequency" for MUI X.
Maybe a link to an X space?

Reply:
I would like to defend that 12 months sounds be our target, not 18. If hardware companies at scale like Apple can pull it off, we should be able to do something similar, even if t's only to deprecate and remove little APIs.